scripts/build_erddap_catalog.py

```diff
@@ -287,6 +287,16 @@ def get_first_nc_file(root):
             return os.path.join(root, content)
 
 
+def get_latest_nc_file(root):
+    '''
+    Returns the lastest netCDF file found in the directory
+
+    :param str root: Root of the directory to scan
+    '''
+    list_of_files = glob.glob('{}/*.nc'.format(root))
+    return max(list_of_files, key=os.path.getctime)
```

Comment:
Should check for empty here and return None if empty -- max on an empty collection will raise a `ValueError`.  It's not uncommon to have deployments created which don't contain files for one reason or another.  See below excerpt.   Thankfully this is handled by exception handlers and the datasets.xml generating scripts continue, but it'd be good to handle the empty value case.

```
[2020-08-14 20:42:00,214 | ERROR]  Error: user: ciglr                                                                                                                                                                                         
Traceback (most recent call last):                                                                                                                                                                                                            
  File "/home/glider/glider-dac/scripts/build_erddap_catalog.py", line 53, in build_erddap_catalog                                                                                                                                            
    f.write(build_erddap_catalog_fragment(data_root, user, deployment, template_dir, root_dir, mode=erddap_name))                                                                                                                             
  File "/home/glider/glider-dac/scripts/build_erddap_catalog.py", line 125, in build_erddap_catalog_fragment                                                                                                                                  
    nc_file = get_latest_nc_file(dir_path)                                                                                                                                                                                                    
  File "/home/glider/glider-dac/scripts/build_erddap_catalog.py", line 297, in get_latest_nc_file                                                                                                                                             
    return max(list_of_files, key=os.path.getctime)
ValueError: max() arg is an empty sequence
[2020-08-14 20:42:00,214 | INFO]  Building ERDDAP catalog fragment for ciglr/burbot-20180726T1435-delayed
[2020-08-14 20:42:00,227 | ERROR]  Error: user: ciglr
Traceback (most recent call last):
...
```